report_generator/data/default.ini

```diff
@@ -1,12 +1,12 @@
 [ATTENDEE]
 # session for attendee csv
-year = 2019
+year = 2020
 # columns
 # comma for mutiple input files
 # indivisual, reserverd, corporate
-paid_date = 3, 3, 3
-nationality = 11, 13, 15
-gender = 12, 14, 16
-job_title = 10, 12, 14
-seniority = 8, 9, 11
-interested_field = 9, 10, 12
+paid_date =         8,  8,  8
+nationality =      16, 16, 19
+gender =           18, 18, 24
+job_title =        15, 15, 18
+seniority =        12, 12, 15
+interested_field = 13, 13, 16
```

Comment:
I'm not particular familiar with how `ini` style should be. But if we're to following this format, we should apply it to all the ini files.

Reply:
+1 from software engineering point of view. However, I will leave the unification to reformat ini for past years to another pull request instead to keep this pull request focusing on 2020 only.